Sources/ConfigCat/ConfigCatClient.swift

```diff
@@ -229,8 +230,10 @@ public final class ConfigCatClient: NSObject, ConfigCatClientProtocol {
                 return
             }
             guard let setting = result.settings[key] else {
-                let message = String(format: "Value not found for key '%@'. Here are the available keys: %@", key, [String](result.settings.keys))
-                self.log.error(message: message)
+                let message = String(format: "Failed to evaluate setting '%@' (the key was not found in config JSON). "
+                    + "Returning the `defaultValue` parameter that you specified in your application: '%@'. Available keys: %@.",
```

Comment:
Available keys should be surrounded with single quotes like `'Key1', 'Key2', 'Key3'`